pubspec.yaml

```diff
@@ -24,6 +24,7 @@ dependencies:
   qr_flutter: ^4.0.0
   webview_flutter_plus: ^0.3.0
   flutter_native_splash: ^2.2.15
+  url_launcher: ^6.1.7
```

Comment:
🙃 It is weird to me to have a separate commit that just adds a dependency (I assume it isn't actually used given the commit title).

Reply:
it is used to launch the browser from the app (showing you the transaction in the mempool), but it also came with a lot of generated code, so I figured I put into a separate commit, so you can safely ignore that for your review. Do you prefer having that all in one commit?